src/App.vue

```diff
@@ -199,8 +199,9 @@
               cols="12"
               lg="3"
             >
-              <!-- Render fee summary only after data is loaded. -->
-              <aside v-if="haveData">
+              <!-- Render fee summary only after data is loaded, -->
+              <!-- and if this isn't a change-requested filing. -->
+              <aside v-if="haveData && getFilingStatus !== FilingStatus.CHANGE_REQUESTED">
```

Comment:
Note: normally I frown upon negative logic such as here ("only show if not X") because it will cause a bug if, say, we add a status and we don't automatically want to show this -- if we specified the exact conditions under which to show the Fee Summary, this would not cause such a bug.

In this case, I'm on the fence. This is an extra condition that applies only to filings in Change-Requested status and today this is pretty clear what it's doing. But, I don't mind changing this to `if status === DRAFT` if you think that's more appropriate.

What do you think?

Reply:
regardless of these options if a new status introduced we may have to change this. so i am good with this

Reply:
I think this is more clear